core/scheduler/service/replay_worker.go

```diff
@@ -98,6 +99,18 @@ func (w *ReplayWorker) Execute(replayID uuid.UUID, jobTenant tenant.Tenant, jobN
 	}
 }
 
+func (w *ReplayWorker) SyncStatus(ctx context.Context, replayWithRun *scheduler.ReplayWithRun, jobCron *cron.ScheduleSpec) (*scheduler.JobRunStatusList, error) {
```

Comment:
```suggestion
func (w *ReplayWorker) SyncStatus(ctx context.Context, replayWithRun *scheduler.ReplayWithRun, jobCron *cron.ScheduleSpec) (scheduler.JobRunStatusList, error) {
```
minor, how about returning value instead of pointer? So it doesn't have to be dereferenced in other function calls using the returned value

Reply:
okay , we can do that as well